languages/go/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# Usage: ./build.sh [--release]
+
+check_command() {
+  if ! command -v "$1" &>/dev/null; then
+    printf '%s\n' "$1 is required to build locally. Please install $2."
+    exit 1
+  fi
+}
+
+check_command cargo Rust
+check_command go Go
+check_command npm Node.js
```

Comment:
We generally presume the environment is correctly bootstraped. These checks can provide a false confidence since we don't validate the version of the scripts.